python/src/etos_api/routers/v0/router.py

```diff
@@ -20,57 +20,76 @@
 from kubernetes import client
 from opentelemetry import trace
 from opentelemetry.trace import Span
-import requests
 
 from etos_api.library.environment import Configuration, configure_testrun
 from etos_api.library.utilities import sync_to_async
-from etos_api.library.validator import SuiteValidator
-from etos_api.routers.lib.kubernetes import namespace
 
 from .schemas import AbortEtosResponse, StartEtosRequest, StartEtosResponse
-from .utilities import wait_for_artifact_created
-
-ROUTER = APIRouter()
+from .utilities import wait_for_artifact_created, validate_suite
+
+ETOSv0 = FastAPI(
+    title="ETOS",
+    version="v0",
+    summary="API endpoints for ETOS v0 - I.e. the version before versions",
+    root_path_in_servers=False,
+)
 TRACER = trace.get_tracer("etos_api.routers.etos.router")
 LOGGER = logging.getLogger(__name__)
 logging.getLogger("pika").setLevel(logging.WARNING)
 
 
-async def download_suite(test_suite_url: str) -> dict:
-    """Attempt to download suite.
+@ETOSv0.post("/etos", tags=["etos"], response_model=StartEtosResponse)
+async def start_etos(etos: StartEtosRequest):
+    """Start ETOS execution on post.
 
-    :param test_suite_url: URL to test suite to download.
-    :return: Downloaded test suite as JSON.
+    :param etos: ETOS pydantic model.
+    :type etos: :obj:`etos_api.routers.etos.schemas.StartEtosRequest`
+    :return: JSON dictionary with response.
+    :rtype: dict
     """
-    try:
-        suite = requests.get(test_suite_url, timeout=60)
-        suite.raise_for_status()
-    except Exception as exception:  # pylint:disable=broad-except
-        raise AssertionError(f"Unable to download suite from {test_suite_url}") from exception
-    return suite.json()
+    with TRACER.start_as_current_span("start-etos") as span:
+        return await _start(etos, span)
 
 
-async def validate_suite(test_suite_url: str) -> None:
-    """Validate the ETOS test suite through the SuiteValidator.
+@ETOSv0.delete("/etos/{suite_id}", tags=["etos"], response_model=AbortEtosResponse)
+async def abort_etos(suite_id: str):
+    """Abort ETOS execution on delete.
 
-    :param test_suite_url: The URL to the test suite to validate.
+    :param suite_id: ETOS suite id
+    :type suite_id: str
+    :return: JSON dictionary with response.
+    :rtype: dict
     """
-    span = trace.get_current_span()
+    with TRACER.start_as_current_span("abort-etos"):
+        return await _abort(suite_id)
 
-    try:
-        test_suite = await download_suite(test_suite_url)
-        await SuiteValidator().validate(test_suite)
-    except AssertionError as exception:
-        LOGGER.error("Test suite validation failed!")
-        LOGGER.error(exception)
-        span.add_event("Test suite validation failed")
-        raise HTTPException(
-            status_code=400, detail=f"Test suite validation failed. {exception}"
-        ) from exception
+
+@ETOSv0.get("/ping", tags=["etos"], status_code=204)
```

Comment:
why is this changed in v0?

Reply:
There is a comment about this on the other function, isnt that sufficient?

Reply:
It is implemented in each version because I want it to do verifications as well. v0 does nothing, but I want newer versions to actually check a few things, such as the version of the ETOS client etc.